src/app/hackernews-api.service.ts

```diff
@@ -25,4 +25,9 @@ export class HackerNewsAPIService {
   		return this.http.get(`${this.baseUrl}/user/${id}`)
   					.map(response => response.json());
  	}
+
+	 fetchPollResult(id: number): Observable<any> {
+		 return this.http.get(`${this.baseUrl}/item/${id}`)
```

Comment:
Since this is essentially the same endpoint as `fetchComments`, we can remove this and only keep the one method. But let's rename it to something more clear since it's not just comments it's also used to retrieve polls as well. Maybe `fetchItemContents`?